.github/workflows/find-reviewer.yml

```diff
@@ -0,0 +1,89 @@
+name: Find Reviewer
+
+on:
+  repository_dispatch:
+    types: [find-reviewer]
```

Comment:
Does this mean this workflow is called each time a `find-reviewer` webhooks is received? Can you help me understand how this works? thx :)

Reply:
Exact. Here are some resources about it you could find useful:
- https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#repository_dispatch
- https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#create-a-repository-dispatch-event

Reply:
see https://github.com/carsonbot/carsonbot/pull/113/files#diff-761989cba708b0fb2f270ad4d799a7a955b1c611d9cefa6fde84611e03e516c0R36

The corresponding code was added in a previous PR, which is indeed weird.

Reply:
So I've defined a special event id called `find-reviewer`. In the `GithubPullRequestApi::findReviewer()`  that @stof linked, I am dispatching this special event. 

It is basically me saying: "Hey start the specific CI job called `find-reviewer`".